### PR TITLE
Thread down airgap flag, mostly for secondary nodes

### PR DESCRIPTION
**Pull request by evanphx**
Goes along with https://github.com/hashicorp/ptfe/pull/33 to be sure that all nodes know they're in airgap mode.

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -61,6 +61,8 @@ data "template_file" "cloud_config" {
     weave_cidr           = "${var.weave_cidr}"
     repl_cidr            = "${var.repl_cidr}"
 
+    network_type = "${var.app_network_type}"
```
Comment (amy-hashi) [inline config.tf]: Where is app_network_type defined?

Comment (evanphx) [reply config.tf]: It's a local, defined up in this file.

Comment (evanphx) [reply config.tf]: It's not a var, just fixed that.

**Comment by evanphx** [comment]
A later PR wired up the need to pass the airgap-installer to the secondaries (because they have to install that data as well) that means this change isn't needed.

**Pull request closed without merging**